### DISCUSSION TITLE
[NoBug] Is this still a bug:  https://github.com/cocos2d/cocos2d-x/issues/20715?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1141] [Q&A] [answered]
Only as reminder for check later:
Is this still a bug: https://github.com/cocos2d/cocos2d-x/issues/20715?

Maybe somebody has time to check?

**Comment by rh101** [comment, marked as answer]
I can't reproduce this problem with Axmol using the provided code.

With `maskedVeil->setInverted(true);`
![image](https://user-images.githubusercontent.com/8603230/227977471-0eb8722b-d4b8-48cd-a59a-884b46f5626c.png)

With `maskedVeil->setInverted(false);`
![image](https://user-images.githubusercontent.com/8603230/227977675-1fd30506-80ff-44b2-908d-cca0efb55b7f.png)


Both results are as expected.

**Reply by aismann** [reply]
@rh101 
Thanks for your check.
